src/pages/ConvertVrt/Convert/index.tsx

```diff
@@ -0,0 +1,204 @@
+const formatI18nextRelativetimeValues = (
+  vrtConversionEndTime: string | undefined,
+): FormatI18nextRelativetimeValuesReturn => {
+  if (vrtConversionEndTime === undefined) {
+    return {
+      realtiveTimeFormatValues: { count: PLACEHOLDER_KEY },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeMissing',
+    };
+  }
+  const MINUTE_IN_MILISECONDS = 60 * 1000;
+  const HOUR_IN_MILISECONDS = 60 * MINUTE_IN_MILISECONDS;
+  const DAY_IN_MILISECONDS = 24 * HOUR_IN_MILISECONDS;
+  const vestingTimeRemainingMs = new Date().getTime() - +vrtConversionEndTime * 1000;
+  let relativeTimeValues: FormatI18nextRelativetimeValuesReturn = {
+    realtiveTimeFormatValues: { count: Math.floor(vestingTimeRemainingMs / DAY_IN_MILISECONDS) },
+    relativeTimeTranslationKey: 'convertVrt.remainingTimeDays',
+  };
+  if (vestingTimeRemainingMs === HOUR_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: { count: Math.floor(vestingTimeRemainingMs / HOUR_IN_MILISECONDS) },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHours',
+    };
+  } else if (vestingTimeRemainingMs < HOUR_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: {
+        count: Math.floor(vestingTimeRemainingMs / MINUTE_IN_MILISECONDS),
+      },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeMinutes',
+    };
+  } else if (vestingTimeRemainingMs < DAY_IN_MILISECONDS) {
+    relativeTimeValues = {
+      realtiveTimeFormatValues: {
+        hours: Math.floor(vestingTimeRemainingMs / HOUR_IN_MILISECONDS),
+        minutes: Math.floor(
+          ((vestingTimeRemainingMs % HOUR_IN_MILISECONDS) / HOUR_IN_MILISECONDS) * 60,
+        ),
+      },
+      relativeTimeTranslationKey: 'convertVrt.remainingTimeHoursAndMinutes',
+    };
+  }
+  return relativeTimeValues;
+};
+
+const Convert: React.FC<ConvertProps> = ({
+  xvsTotalWei,
+  xvsToVrtRate,
+  vrtLimitUsedWei,
+  vrtLimitWei,
+  vrtConversionEndTime,
+}) => {
+  const styles = useStyles();
+  const { t, Trans } = useTranslation();
+  const readableXvsAvailable = useConvertToReadableCoinString({
+    valueWei: xvsTotalWei,
+    tokenId: XVS_ID,
+  });
+  const { relativeTimeTranslationKey, realtiveTimeFormatValues } =
+    formatI18nextRelativetimeValues(vrtConversionEndTime);
+
+  return (
+    <div css={styles.root}>
+      <section css={styles.title}>
+        <Typography variant="h3">{readableXvsAvailable}</Typography>
+        <Typography variant="small2">{t('convertVrt.xvsAVailable')}</Typography>
+      </section>
+      <ConnectWallet message={t('convertVrt.connectWalletToConvertVrtToXvs')}>
+        <AmountForm onSubmit={noop} maxAmount={vrtLimitWei.toFixed()}>
+          {({ values }) => {
+            const xvsValue = useMemo(() => {
+              if (values.amount) {
+                return new BigNumber(values.amount).times(xvsToVrtRate).toFixed();
+              }
+              return '';
+            }, [values.amount, xvsToVrtRate]);
```

Comment:
There's no point in memoizing this value here, since `values.amount` will change very often you'll actually lengthen the initial rendering time without making subsequent renders faster (given most of the re-renders will come from `values.amount` changing).
You could create a `convertToXvsTokens` (or any name you see fit) function outside of the `return` block and that uses `useCallback` to return a memoized function that accepts the amount as parameter:
```typescript
const convertToXvsTokens = React.useCallback((xvsAmountTokens: string | undefined) =>
  values.amount ?
    new BigNumber(values.amount).times(xvsToVrtRate).dp(VRT_DECIMALS).toFixed() : '', [xvsToVrtRate]);
```

Please note we also should apply format the final value so it doesn't contain more decimals than XVS has.